analyzer/vulnerability-statements-processor/src/test/java/eu/fasten/vulnerabilitystatementsprocessor/VulnerabilityStatementsProcessorTest.java

```diff
@@ -30,13 +30,7 @@
 import org.mockito.Mockito;
 
 import java.text.ParseException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.LinkedHashSet;
-import java.util.Map;
+import java.util.*;
```

Comment:
Could you please check which IntelliJ auto-formatting rule is responsible for this change? Please adopt this setting it and make it available for everybody on the repository level, this is usually easily possible in Jetbrains producs by placing the config in the right place. (See https://www.jetbrains.com/help/idea/configuring-code-style.html and look for "project level).

Reply:
Addressed in b06f236